README.md

```diff
@@ -26,6 +26,7 @@ These reside in the [unit features directory](features/unit)
 | @unit.applications.boxes             | Box endpoints for Algod and Indexer.                                                                        |
 | @unit.atomic_transaction_composer    | ABI / atomic transaction construction unit tests.                                                           |
 | @unit.atc_method_args                | Test that algod's Atomic Transaction Composer assserts that the same number of arguments given as expected. |
+| @unit.client-no-headers              | Tests that clients don't add non-standard http headers when no (or an empty) auth token is provided.        |
```

Comment:
Should this be Javascript only? If so, I recommend adding a row to the [not full supported table](https://github.com/algorand/algorand-sdk-testing/blob/master/README.md?plain=1#L83)